torch/_inductor/kernel/bmm.py

```diff
@@ -87,6 +89,38 @@ def bmm_grid(b, m, n, meta):
 
 @register_lowering(aten.bmm)
 def tuned_bmm(mat1, mat2, *, layout=None):
+    if all(x.get_device().type == "cpu" for x in [mat1, mat2]):
+
+        def is_valid_to_require_contiguous(t):
+            if not ir.is_storage_and_layout(t):
+                return True
+            _, layout = ir.as_storage_and_layout(t, freeze=False)
+            return not isinstance(layout, ir.FixedLayout)
```

Comment:
why not checking `ir.FlexiableLyout` directly?

Reply:
Modified to `ir.FlexiableLyout`.